exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,10 @@
-var numberOfStudents = 15;
-var numberOfMentors = 8;
+let numberOfStudents = 15;
+let numberOfMentors = 8;
+let totalpersons = numberOfStudents + numberOfMentors;
+console.log(
+  `Percentage students: ${Math.round((numberOfStudents * 100) / totalpersons)}`
```

Comment:
Since you are using lambda functions already, should we create a simple one to extract calculation logic in this exercise?